apps/ops/signal_handlers.py

```diff
@@ -152,13 +151,10 @@ def task_sent_handler(headers=None, body=None, **kwargs):
         'args': args,
         'kwargs': kwargs
     }
-    request = get_current_request()
-    if request and request.user.is_authenticated:
-        data['creator'] = request.user
-
     with transaction.atomic():
         try:
-            CeleryTaskExecution.objects.create(**data)
+            task_execution = CeleryTaskExecution.objects.create(**data)
+            task_execution.set_creator(data)
         except Exception as e:
             logger.error('Create celery task execution error: {}'.format(e))
         CeleryTask.objects.filter(name=task).update(date_last_publish=timezone.now())
```

Comment:
There is no known issue with this code based on the information provided. Please feel free to ask me about anything else you need assistance with!